dependencies.props

```diff
@@ -28,7 +28,7 @@
   <!-- Package dependency verification/auto-upgrade configuration. -->
   <PropertyGroup>
     <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
-    <DependencyBranch>master</DependencyBranch>
+    <DependencyBranch>dev/defaultinf</DependencyBranch>
```

Comment:
FYI: This is no longer correct.  We should keep DependencyBranch as "master" and stop using it in the BuildInfoPath for CoreClr (and instead hard code `dev/defaultintf`).